src/main/java/org/jnode/fs/xfs/Superblock.java

```diff
@@ -114,9 +126,7 @@ public int getVersion() {
      *
      * @return the inode size.
      */
-    public int getInodeSize() {
-        return getUInt16(0x68);
-    }
+    public int getInodeSize() { return getUInt16(0x68);}
```

Comment:
```suggestion
    public int getInodeSize() { 
        return getUInt16(0x68);
    }
```

Reply:
The suggestion was applied, thanks Erwin.